### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Trojan295**
Added a flow diagram of for the OIDC plugin and OIDC provider
interaction.
Added configuration parameter table.
Added example request and responses.

Solves #19

You can check it out here:
https://github.com/Trojan295/kong-oidc/blob/issue-15/README.md

@phirvone, @jeremyjpj0916, @Pinguwien, @tsyrjanen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +55,95 @@ already posses acces token to go through kong. The actual token verification is
...
+
+To enable the plugin only for one API:
+
+```
+POST /apis/<api_id>/plugins/ HTTP/1.1
+Host: localhost:8001
+Content-Type: application/x-www-form-urlencoded
+Cache-Control: no-cache
+
+name=oidc&config.client_id=kong-oidc&config.client_secret=29d98bf7-168c-4874-b8e9-9ba5e7382fa0&config.discovery=http%3A%2F%2Fkeycloak%3A8080%2Fauth%2Frealms%2Fmaster%2F.well-known%2Fopenid-configuration
```
Comment (tsyrjanen) [inline README.md]: Better to say oidc_provider and not keycloak (this is in 3 places).

Comment (Trojan295) [reply README.md]: In fact this is an example query, so it don't thing it's required to generalize it.

Does `https%3A%2F%2F<oidc_provider>%2F.well-known%2Fopenid-configuration` look better?

Comment (tsyrjanen) [reply README.md]: Yes. Point is that the oidc provider could be something else than keycloak.

Comment (Trojan295) [reply README.md]: OK, I changed it. Can you later squash the commits when merging the PR?

**Pull request merged**